core-bundle/assets/controllers/check-all-controller.js

```diff
@@ -3,7 +3,61 @@ import { Controller } from '@hotwired/stimulus';
 export default class extends Controller {
     static targets = ['source', 'input'];
 
-    update() {
+    static values = {
+        rowGuards: {
+            type: Array,
+            default: ['input', 'label', 'button', 'a', '.operations'],
+        },
+    };
+
+    initialize() {
+        this.start = null;
+    }
+
+    #shiftToggle(el) {
```

Comment:
private methos (at least in PHP) are usually after the public method (but we don't have a CS fixer for that afaik)

Reply:
Old habit of mine as there is no real styleguide for that (not even in airbnb). This can be changed however.

Reply:
See 439d7688c5babaa32636b5d0f30f04d936dcb0ca